lib/bezier.js

```diff
@@ -1091,8 +1091,15 @@
           self=this;
       return utils.roots(this.points, line).filter(function(t) {
         var p = self.get(t);
-        return (mx <= p.x && p.x <= MX && my <= p.y && p.y <= MY);
+        return between(p.x, mx, MX) && between(p.y, my, MY);
       });
+      function between(x, m, M){ 
+        return (m <= x && x <= M) || fEq(x, m) || fEq(x, M); 
+      }
+      function fEq(a,b,precision) { 
```

Comment:
can we rename this to a more descriptive function name? maybe `approx` or something?